2_adventures/001/jimmylorunning/01_day1/README.md

```diff
@@ -0,0 +1,13 @@
+- `git checkout master`
+- `git remote add upstream https://github.com/RubySteps/21-day-challenge`
+- `git remote -v` (to make sure it's there)
+- `git pull upstream master` (to get new changes since I last pulled)
+- `git checkout -b 01_day1`
+- `mkdir -p 2_adventures/001/jimmylorunning/01_day1/'`
+- `vim !$README.md`
```

Comment:
Yes! I should've explained. It's a nice trick :). Also !!:x where x is the number of the argument you want... ex:

$ cp hello.txt goodbye.txt
$ vim !!:1  => expands to vim hello.txt
$ vim !!:2  => expands to vim goodbye.txt